### PR TITLE
Log mining tweaks

### PR DESCRIPTION
**Pull request by gpolitis**

### CODE REVIEW

**Comment by gpolitis** [comment]
@bgrozev or @bbaldino could you please take a look?

File: src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java [hunk 1]
```diff
@@ -462,12 +462,11 @@ public void incomingPacketInfo(
         if (logger.isTraceEnabled())
         {
             logger.trace(diagnosticContext
-                .makeTimeSeriesPoint("incoming_packet", nowMs)
+                .makeTimeSeriesPoint("in_pkt", nowMs)
```
Comment (bbaldino) [inline src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java]: just wondering, what's the reason for the shorter name>

File: src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java [hunk 2]
```diff
@@ -462,12 +462,11 @@ public void incomingPacketInfo(
...
                 .addKey("rbe", hashCode())
-                .addField("arrival_time_ms", arrivalTimeMs)
-                .addField("send_time_24bits", sendTime24bits)
-                .addField("send_time_ms", sendTimeMs)
-                .addField("payload_size", payloadSize)
+                .addField("recv", arrivalTimeMs)
+                .addField("send", sendTimeMs)
```
Comment (bbaldino) [inline src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java]: `incoming_packet` -> `in_pkt` didn't seem too bad, but some of these shorter names add a lot of ambiguity.

File: src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java (hunk 1 above)
Comment (gpolitis) [reply src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java]: The motivation here is to save some bytes.. These are the packet trace logs that need to be captured so I'm trying to save whatever I can in terms of size.

File: src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java (hunk 2 above)
Comment (gpolitis) [reply src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java]: I agree, maybe I could add some clarifying suffixes, like '_ms' in the send/recv fields?

File: src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java (hunk 1 above)
Comment (bbaldino) [reply src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java]: ok, i think the way they are now is a bit ambiguous, in fact i'd missed here that this was referring to a timestamp.  i'd say keep them legible enough that there isn't ambiguity, or, go all the way and make them totally encoded (and short) and create a script or something that will be used to parse them?

File: src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java [hunk 3]
```diff
@@ -462,12 +462,11 @@ public void incomingPacketInfo(
         if (logger.isTraceEnabled())
         {
             logger.trace(diagnosticContext
-                .makeTimeSeriesPoint("incoming_packet", nowMs)
+                .makeTimeSeriesPoint("in_pkt", nowMs)
                 .addKey("rbe", hashCode())
```
Comment (bbaldino) [inline src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java]: for example, this as is is tough...i assumed it's 'this is the hash of this remote bandwidth estimator instance' ? so maybe `rbe_id` would be better?

File: src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java (hunk 1 above)
Comment (bbaldino) [reply src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java]: i could see this be `in_pkt_ts`

File: src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java [hunk 4]
```diff
@@ -462,12 +462,11 @@ public void incomingPacketInfo(
...
-                .addField("arrival_time_ms", arrivalTimeMs)
-                .addField("send_time_24bits", sendTime24bits)
-                .addField("send_time_ms", sendTimeMs)
-                .addField("payload_size", payloadSize)
+                .addField("recv", arrivalTimeMs)
```
Comment (bbaldino) [inline src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java]: `recv_time_ms`?

File: src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java (hunk 2 above)
Comment (bbaldino) [reply src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java]: `send_ts_ms`?

File: src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java [hunk 5]
```diff
@@ -462,12 +462,11 @@ public void incomingPacketInfo(
...
+                .addField("send", sendTimeMs)
+                .addField("size", payloadSize)
```
Comment (bbaldino) [inline src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java]: `pkt_size_bytes`?

File: src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java (hunk 1 above)
Comment (gpolitis) [reply src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java]: I think you had it right, in_pkt is the name of the measurement. The second parameter sets the timestamp of the measurement.

**Comment by gpolitis** [comment]
test this please

**Comment by gpolitis** [comment]
@bbaldino could you please merge this if you have no further comments?

**Pull request merged**